src/chromium/interoperability-with-cpp/example-bindings.md

```diff
@@ -14,10 +14,8 @@ Point out:
 * Native support for Rust slices in C++
 * Calls from C++ to Rust, and Rust types (in the top part)
 * Calls from Rust to C++, and C++ types (in the bottom part)
-* If the function definitions in C++ or Rust don't match the cxx::bridge,
-  a compilation failure results.
```

Comment:
The point about compilation failures is now covered in the speaker notes for the *previous* slide.